build.rs

```diff
@@ -26,8 +23,7 @@ fn main() {
         cfg.file(if msvc {"src/arch/i686.asm"} else {"src/arch/i686.S"});
         cfg.define("X86", None);
     } else {
-        panic!("\n\nusing currently unsupported target triple with \
-                stacker: {}\n\n", target);
+        cfg.file("fallback.rs");
```

Comment:
I think this is passing a Rust file to a C compiler?

Reply:
"it works" :laughing: I wasn't sure either and just tried it. I can add a travis builder for it. I'm guessing the `gcc` crate magically knows to invoke rustc?

Reply:
Hm I think this case just isn't hit on CI? If you comment out the above coditions and fall through to this I think you'll probably see some errors in one form or another

Reply:
You're right. I thought I was choosing my targets correctly, but I forgot that it never gets linked unless I run `cargo test`. It built, it just didn't link.

I fixed it by conditionally including the rs file as a module in the fallback case